pkg/controller/import-controller.go

```diff
@@ -243,12 +254,12 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 			anno[AnnRequiresScratch] = "true"
 		} else {
 			r.recorder.Event(pvc, corev1.EventTypeWarning, ErrImportFailedPVC, pod.Status.ContainerStatuses[0].LastTerminationState.Terminated.Message)
+			anno[AnnRunningCondition] = "false"
+			anno[AnnRunningConditionMessage] = pod.Status.ContainerStatuses[0].LastTerminationState.Terminated.Message
+			anno[AnnRunningConditionReason] = pod.Status.ContainerStatuses[0].LastTerminationState.Terminated.Reason
 		}
 	}
 
-	if pod.Status.ContainerStatuses != nil {
-		anno[AnnPodRestarts] = strconv.Itoa(int(pod.Status.ContainerStatuses[0].RestartCount))
-	}
```

Comment:
Why was this deleted?

Reply:
its part of setConditionFromPod now

Reply:
Which is probably not a good plan, so reverting that.